oak_runtime/src/tests.rs

```diff
@@ -440,80 +454,9 @@ fn create_node_invalid_configuration_err() {
     );
 }
 
-/// Create a test Node that creates a Node with a less confidential label and fails.
-///
-/// If this succeeded, it would be a violation of information flow control, since the original
-/// confidential Node would be able to spawn "less confidential / public" Nodes and use their side
-/// effects as a covert channel to exfiltrate confidential data.
-#[test]
-fn create_node_less_confidential_label_err() {
```

Comment:
I agree that we cannot test this at the moment because we can only create nodes from public nodes. It would be posible to test the same logic by tring to create a more trusted node from a public untrusted node (although I know that we don't have full support for integrity labels yet I think there is enough to actually test this logic).

Reply:
I had a go, but unfortunately it's not straightforward &ndash; the tests currently use a logging pseudo-Node as the object to be created, but that can only be created with a `public_untrusted` label anyway (since #1688).  Other types of Node also fail to be (easily) created from a test:
 - Wasm Nodes fail to be created because there isn't a Wasm module available
 - gRPC pseudo-Nodes fail to be created because there isn't gRPC config available

So I agree it should be possible to put something in to replace this test, but it's a bit more involved than this PR.

Reply:
Yes, I thought that it would fall back on the default root TLS certs, but I forgot that they are in the loader, not the runtime.

Reply:
I've added another commit reinstating a couple of variant tests, but I'm not completely happy with the `include_bytes!("../../examples/certs/local/local.pem")` stuff &ndash; WDYT?

Reply:
I think using `include_str! in the tests is fine.

The other option I thought about was to create a dummy node type specifically for testing that can be created with empty configuration (something that we can safely mark as sandboxed because it doesn't ever to anything other than close the initial handle it receives and therefore does not have the restrictions on privilege). This feels a bit messy, seeing that it will be visible to developers as a possible node configuration but will probably only ever be useful in the runtime internally for testing.

On balance, I prefer the option you currently implemented.